moped-editor/src/components/ExternalLink.js

```diff
@@ -14,7 +26,11 @@ const ExternalLink = ({ url, text, linkColor, underline }) => {
           underline={underline ?? "hover"}
         >
           {text}
-          <OpenInNewIcon style={{ fontSize: 16 }} />
```

Comment:
Thanks for moving this style directive out of the JSX and back into the component.